source/ShipInfoPanel.cpp

```diff
@@ -390,10 +391,18 @@ void ShipInfoPanel::DrawOutfits(const Rectangle &bounds, Rectangle &cargoBounds)
 	for(const auto &cat : GameData::GetCategory(CategoryType::OUTFIT))
 	{
 		const string &category = cat.Name();
+		if(category.empty())
+			continue;
 		auto it = outfits.find(category);
 		if(it == outfits.end())
 			continue;
 
+		auto validOutfits = std::ranges::filter_view(it->second,
+			[](const Outfit *o){return o->IsDefined() && !o->DisplayName().empty();});
```

Comment:
There should also be spaces at the beginning and end of the lambda body.
```suggestion
			[](const Outfit *o){ return o->IsDefined() && !o->DisplayName().empty(); });
```